src/library/scala/collection/IterableOnce.scala

```diff
@@ -1071,7 +1071,7 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
 
   def toIndexedSeq: immutable.IndexedSeq[A] = immutable.IndexedSeq.from(this)
 
-  @deprecated("Use Stream.from(it) instead of it.toStream", "2.13.0")
+  @deprecated("Use LazyList.from(it) instead of it.toStream", "2.13.0")
```

Comment:
I think we should recommend using `it.to(LazyList)` instead, like advised here:

https://github.com/scala/scala/blob/250136a603b48d18f341934a0b1ee02aaa8c45f0/src/library/scala/collection/IterableOnce.scala#L142-L143